lib/mapper.js

```diff
@@ -52,6 +49,7 @@ exports.page = function(page, settings){
 
 exports.screen = function(screen, settings){
   var result = createCommonGAForm(screen, settings);
```

Comment:
Sure, but anyone can manually set `context.page`; have you made a screen call with `context.page` set to ensure that it won't stop data flowing into GA? It would be bad if we assumed it wouldn't break GA without actually verifying it. Super edge case but one that would be super frustrating